react/src/hooks/utils.tsx

```diff
@@ -103,7 +103,10 @@ export async function queryTableData<RowData extends object>(
     }
     // page information
     searchParams.append("page", `${query.page}`);
```

Comment:
When page is 0 a `page=0` param is added to the query string, which I don't think has an effect on the request.

Reply:
It doesn't, I guess I can group this under the condition as well.